data-exporter/src/main/java/org/reactome/release/EuropePMC.java

```diff
@@ -0,0 +1,398 @@
+package org.reactome.release;
```

Comment:
I just noticed this now, but the package for everything in this project should probably be `org.reactome.release.dataexport`. Other projects contain all of their code in their own package, such as `org.reactome.release.chebiupdate` or `org.reactome.release.updateDOIs`.

Reply:
No problem -- I'll create the subpackage.